scripts/linters/css_linter.py

```diff
@@ -58,6 +59,34 @@ def all_filepaths(self):
         """Return all filepaths."""
         return self.files_to_lint
 
+    @staticmethod
+    def _get_trimmed_error_messages(css_lint_output):
+        """Remove extra bits from stylelint error messages.
+
+        Args:
+            css_lint_output: str. Output returned by the css linter.
+
+        Returns:
+            str. A string with the trimmed error messages.
+        """
+        error_messages = []
+        # We need to extract messages from the list and split them line by
+        # line so we can loop through them.
+        css_output_lines = css_lint_output.split('\n')
+        for line in css_output_lines:
+            # Stylelint messages starts with line numbers and then a
+            # "x"(\u2716) and a message-id in the end. We are capturing these
+            # and then replacing them with empty string('').
+            if re.search(r'^\d+:\d+', line.lstrip()):
+                unicode_x = re.search(r'\u2716', line).group(0)
+                message_id = re.search(r'(\w+-*)+$', line).group(0)
+                error_message = (
+                    line.replace(unicode_x, '').replace(message_id, ''))
```

Comment:
Does this replace all instances of unicode_x or just the first occurrence?

Also, does this replace all instances of message_id or just the last occurrence?

If the answer is "all" to any of the above, please either fix them to replace just one occurrence, or assert that there is only one occurrence before doing any mutations to the string, so that you don't end up with mangled messages.

Reply:
Changed to replace only one occurrence